server/Mistakes.Journal.Api/Api/Shared/RequestsParameters/SolvedParameters.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Mistakes.Journal.Api.Api.Shared.RequestsParameters
+{
+    public class SolvedParameters : PagingParameters
```

Comment:
Why it extends PagingParameters? It actually has nothing to do with them.

Reply:
so all these parameters should be separate